docs/installation/octopus-in-container/octopus-server-container-windows.md

```diff
@@ -39,17 +41,19 @@ Read the Docker [docs](https://docs.docker.com/engine/reference/commandline/run/
 
 |  Name       |    |
 | ------------- | ------- |
-|**SqlDbConnectionString**|Connection string to the database to use|
-|**MasterKey**|The master key to use to connect to an existing database. If not supplied, and the database does not exist, it will generate a new one. If the database does exist, this is mandatory|
-|**OctopusAdminUsername**|The admin user to create for the Octopus Server|
-|**OctopusAdminPassword**|The password for the admin user for the Octopus Server|
+|**DB_CONNECTION_STRING**|Connection string to the database to use|
+|**MASTER_KEY**|The master key to use to connect to an existing database. If not supplied, and the database does not exist, it will generate a new one. If the database does exist, this is mandatory|
```

Comment:
```suggestion
|**MASTER_KEY**|The master key to use to connect to an existing database. If not supplied, and the database does not exist, it will generate a new one. This is mandatory if the database exists|
```